javascript/output_container.js

```diff
@@ -1,6 +1,6 @@
 var OutputContainer = function(options){
   options = $.extend({}, options)
-  var view         = this.view = $('<span class="selected_text_container" tabindex="0"></span>')
+  var view         = $('<span class="selected_text_container" tabindex="0"></span>')
```

Comment:
Why this change?

Reply:
I added it in the last PR, but I handed noticed that view is already in the public API.